pkg/cloudprovider/providers/azure/azure_blobDiskController.go

```diff
@@ -542,14 +542,6 @@ func (c *BlobDiskController) createStorageAccount(storageAccountName string, sto
 		c.addAccountState(storageAccountName, newAccountState)
 	}
 
-	if !bExist {
-		// SA Accounts takes time to be provisioned
-		// so if this account was just created allow it sometime
-		// before polling
-		glog.V(2).Infof("azureDisk - storage account %s was just created, allowing time before polling status", storageAccountName)
-		time.Sleep(25 * time.Second) // as observed 25 is the average time for SA to be provisioned
```

Comment:
It's surprising we didn't get bugs on this previously. There are definitely cases where this can take longer than 25 seconds, but I guess downstream code probably retries too.

Reply:
point is we already use sync way to wait for storage account creating successfully, it's not necessary to wait any more:
`err := <-errChan`

Reply:
I understand, just surprised this sleep lasted this long without being hardened but I guess it was good enough.